### PR TITLE
Login problem and clickable select association

### PR DESCRIPTION
**Pull request by sarahbadr17**
This PR fixes the login problem: the current user association id is always set to something and thus there's no null problem.
Plus, when you click on the association on select association you can directly go to home.

### CODE REVIEW

File: app/src/androidTest/java/com/github/se/assocify/screens/SelectAssociationScreenTest.kt [hunk 1]
```diff
@@ -78,6 +83,7 @@ class SelectAssociationTest : TestCase(kaspressoBuilder = Kaspresso.Builder.with
 
   @RelaxedMockK lateinit var mockCurrentUser: CurrentUser
 
+  @RelaxedMockK lateinit var mockFirebase: Firebase
```
Comment (SidonieBouthors) [inline app/src/androidTest/java/com/github/se/assocify/screens/SelectAssociationScreenTest.kt:86]: Is it necessary to mockk Firebase given that the API is already mocked? 
Same question for the mockk current user, it doesn't look like you use either of those in the tests.

File: app/src/main/java/com/github/se/assocify/ui/screens/login/LoginViewModel.kt [hunk 2]
```diff
@@ -18,6 +18,7 @@ class LoginViewModel(private val userAPI: UserAPI, private val navActions: Navig
   /** Updates the userId of the UI state */
   fun updateUser() {
     CurrentUser.userUid = getCurrentUser()!!.uid
+    CurrentUser.associationUid = ""
```
Comment (SidonieBouthors) [inline app/src/main/java/com/github/se/assocify/ui/screens/login/LoginViewModel.kt:21]: We'll need to fix this later when we are using Supabase and have the right User/Association linking: make sure to add a task for it in the backlog so we don't forget

**Review by SidonieBouthors — COMMENTED** [review]
Looks good :) Pretty much ready to merge

File: app/src/androidTest/java/com/github/se/assocify/screens/SelectAssociationScreenTest.kt (hunk 1 above)
Comment (sarahbadr17) [reply app/src/androidTest/java/com/github/se/assocify/screens/SelectAssociationScreenTest.kt:86]: Yes, I'll remove it. Thanks!

**Review by SidonieBouthors — APPROVED** [review]
All good!

**Pull request merged**